### DISCUSSION TITLE
Issue with the survey command (due to another plugin)

### DISCUSSION THREAD

**Discussion by moabist** [TownyAdvanced/TownyResources#87] [Q&A] [answered]
Hi, i'm using twony as well as sentinel on my Server and wanted to add this plugin as well. Unfortunately it seems like Sentinel uses /tr as a shorter form for their /trait command which then blocks the /tr survey command since the server thinks the player is attempting to give an NPC a trait instead of doing a survey. 
Therefore i thought i might find some help here since i'd really like to add TownyResources but can't afford to dispense on Sentinels.

**Comment by LlmDl** [comment, marked as answer]
You can use your server's commands.yml to set up aliases.

**Reply by moabist** [reply]
Ok thanks i'll try it
